openapi-integration-tests/src/test/resources/service/return/ballerina/content_schema_has_one_of_type.bal

```diff
@@ -20,6 +15,11 @@ public type PetForm record {
     string lastName?;
 };
 
+public type InlineResponse200Ok record {|
+    *http:Ok;
+    inline_response_200 body;
```

Comment:
Is there a reason not to follow the pascal case convention here?

Reply:
This is a generated name(`inline_response_200`) from the swagger parser when `flatten` option is enabled. We are addressing this in the sanitize option.